packages/react/src/tooltip/root/useTooltipRoot.ts

```diff
@@ -158,7 +158,7 @@ export function useTooltipRoot(params: useTooltipRoot.Parameters): useTooltipRoo
       positionerElement,
       setPositionerElement,
       popupRef,
-      getRootTriggerProps,
+      triggerProps: getRootTriggerProps(),
```

Comment:
This might not work as intended in all cases. Components that use `useButton` need to have `externalProps` passed into the prop getter, as merging props isn't trivial there (event handlers are not chained, but `onClick` can be called from other handlers).

Reply:
Just saw that `mergeProps` can also accept a function. So we should do `mergeProps(getRootTriggerProps, other, getButtonProps)`?